ci/Dockerfile

```diff
@@ -53,16 +52,17 @@ ENV GOPATH=/opt/app-root/go \
     LC_ALL=en_US.UTF-8 \
     LANG=en_US.UTF-8 \
     LANGUAGE=en_US.UTF-8 \
-    NVM_DIR="/root/.nvm" \
+    NVM_DIR="/opt/.nvm" \
     TMPDIR=/tmp \
+    DOTNET_CLI_TELEMETRY_OPTOUT=1 \
     NODE_COMPILE_CACHE="/opt/cdxgen-node-cache" \
     PYTHONPATH=/opt/pypi:${PYTHONPATH} \
     CDXGEN_IN_CONTAINER=true \
-    SDKMAN_DIR=/root/.sdkman \
-    SDKMAN_CANDIDATES_DIR=/root/.sdkman/candidates \
-    DOTNET_CLI_TELEMETRY_OPTOUT=1 \
+    CDXGEN_TEMP_DIR=/tmp/cdxgen-temp \
```

Comment:
Unfortunately, none of my local tests involved writing to this directory. Plus, my nerdctl run command was volume mounting the host's `/tmp` directory with `-v /tmp:/tmp` and therefore was writable somehow 🤔

Reply:
See #1868